tasks/steps/update_instance.yml

```diff
@@ -1,11 +1,20 @@
 ---
 
-- name: 'Update instance'
-  file:
-    src: '{{ instance_info.dist_dir }}'
-    dest: '{{ instance_info.instance_dist_dir }}'
-    owner: '{{ cartridge_app_user }}'
-    group: '{{ cartridge_app_group }}'
-    state: link
-  when: cartridge_multiversion
+- when: cartridge_multiversion
   tags: cartridge-instances
+  block:
+    - name: 'Update instance application'
+      file:
+        src: '{{ instance_info.dist_dir }}'
+        dest: '{{ instance_info.instance_dist_dir }}'
+        owner: '{{ cartridge_app_user }}'
+        group: '{{ cartridge_app_group }}'
+        state: link
+
+    - name: 'Check if instance restart is required to update application'
```

Comment:
'Check if instance restart is required to apply changes'?

Reply:
To apply what changes?